.github/workflows/ci.yml

```diff
@@ -30,6 +30,13 @@ jobs:
           - "-profile test,docker"
           - "-profile test_one_sample,docker"
     steps:
+      - name: Free some space
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+
```

Comment:
This bit clears up certain tools that come along with the github runner thereby leaving more space for the containers used in the pipeline.